.dwollaci.yml

```diff
@@ -19,4 +19,4 @@ stages:
           --account 455620323323 \
           --bucket dwolla-code \
           --stage admin \
```

Comment:
Should the stage be prod rather than admin? I am pretty sure it will not make a difference, but maybe just more logically correct?

Reply:
Idk, I typically don't like to put stuff in Prod that we can't also put in UAT and DevInt.

Reply:
cool thanks, that makes sense. i dont really have a deep understanding of what belongs in "admin".